packages/amazon/src/serverGroup/configure/wizard/pages/volumes/VolumeEditor.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { FormikErrors, FormikProps } from 'formik';
+import { IServerGroupCommandBlockDevice, IVolumeListenerAction, IBlockDevicesCommand } from '../../../serverGroupConfiguration.service';
+import { ConfigureVolumeConfigModal } from './ConfigureVolumeModal';
+import { Application, IWizardPageComponent } from '@spinnaker/core';
+
+export interface IVolumeEditorProps {
+  app: Application;
+  formik: FormikProps<IBlockDevicesCommand>;
+}
+
+export interface IVolumeEditorState {
+  backingModel: IServerGroupCommandBlockDevice[];
+}
+
+export class VolumeEditor extends React.Component<IVolumeEditorProps, IVolumeEditorState>
+  implements IWizardPageComponent<IBlockDevicesCommand> {
+  constructor(props: IVolumeEditorProps) {
+    super(props);
+  }
+
+  public validate(
+    values: IBlockDevicesCommand,
+  ): FormikErrors<IBlockDevicesCommand> {
+
+    const errors = {} as any;
+    const { blockDevices } = values;
+    console.log(blockDevices)
+    return errors;
+  }
+
```

Comment:
I don't think this function gets called ever, and it is also a noop, so might as well remove it